solr/packaging/test/test_extraction.bats

```diff
@@ -18,6 +18,24 @@
 
 load bats_helper
 
+setup_file() {
+  if command -v docker >/dev/null 2>&1 && docker info >/dev/null 2>&1; then
```

Comment:
This use of docker I suspect could be moved to a helper as we add other tests that depend on Docker.